src/main/java/uk/gov/hmcts/reform/hmc/helper/hmi/ListingMapper.java

```diff
@@ -40,7 +40,8 @@ public Listing getListing(HearingDetails hearingDetails) {
             .listingJohs(listingJohsMapper.getListingJohs(hearingDetails.getPanelRequirements()))
             .listingHearingChannels(hearingDetails.getHearingChannels())
             .listingLocations(listingLocationsMapper.getListingLocations(hearingDetails.getHearingLocations()))
-            .amendReasonCode(hearingDetails.getAmendReasonCode())
+            .amendReasonCode(hearingDetails.getAmendReasonCodes() == null ? null :
+                    String.join(",", hearingDetails.getAmendReasonCodes()))
```

Comment:
Perhaps now we're moving to an array it'd be better to make this a bit less confusing and just check in here whether the list is non-empty, and if so then fix to `Constants.AMEND_REASON_CODE` (and delete the pre-existing confusing override in `HmiSubmitHearingRequestMapper`).

Reply:
Fixed in https://github.com/hmcts/hmc-cft-hearing-service/pull/242/commits/9060477b00566402258396dd97f0ec1ca4e20604